lfs/transfer_queue.go

```diff
@@ -125,11 +127,17 @@ func (q *TransferQueue) processBatch() {
 
 	objects, err := Batch(transfers)
 	if err != nil {
-		q.errorc <- err
-		sendApiEvent(apiEventFail)
-		return
+		if isNotImplError(err) {
+			tracerx.Printf("queue: batch not implemented, disabling")
+			configFile := filepath.Join(LocalWorkingDir, ".gitconfig")
+			git.Config.UnsetLocalKey(configFile, "lfs.batch")
```

Comment:
Makes sense, though I'm thinking that messing with `.gitconfig` is troublesome because now that file is modified and needs to be committed and pushed. I think we should set it in `.git/config`. That won't propagate, of course, but it'll get set in each repo on the first attempt.